### DISCUSSION TITLE
現在の「質問」の「意見交換」について

### DISCUSSION THREAD

**Discussion by yoshi389111** [increments/qiita-discussions#445] [要望・アイデア・不具合]
現在の Qiita では、「質問」のフォーマットに「Q&A」と「意見交換」というものがあると思います。
この「意見交換」は「質問」の一種ではなく、「記事」の一種にしたほうが良いのではないかと思いました。
あるいは、「質問」にも「記事」にも「意見交換」というフォーマットがあってもよいのかもしれませんが。

あくまで個人的なイメージですが、意見交換をしたい場合には、「質問」よりも「記事」に投稿したほうが、比較的多くの意見が集まるような気がしています。
（きっと、閲覧数は「質問」よりも、「記事」の方が多いだろうと思っています）

現状は「記事」で意見交換をしたいとなると、本文で意見を募集して、意見交換はコメント欄で、となると思いますが、間に関連記事や広告が挟まり、少し使いにくいような気がしています。

**Comment by tanaka-asami415** [comment]
@yoshi389111 
ありがとうございます🙏
参考にさせていただきます！

**Comment by blue32a** [comment]
意見交換は別の機能にしてはどうか、という点については同意します。

現状だと質問/意見交換の使い分けなどが示されておらず、質問者としても回答者としても使い方を迷うところです。

また、私はよく質問をみていますが、意見交換はとても少なく、見つけても活発な意見交換がされているというイメージもありません。
なので質問一本でも十分ではないかと思います。（他の人もコメントできるようにするならなおさら #447）

恐らく質問とは期待するものが異なるのだと思うので、別の切り口を検討していかがでしょうか？
例えば、期間ごとに特定のテーマを設ける、投票機能を付けるなど。
